src/services/api/index.ts

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: process.env.SERVER_URL,
```

Comment:
This env really works? we're using vite, and work with environment variables is a quite different
check it out: https://vitejs.dev/guide/env-and-mode.html